sdcm/tester.py

```diff
@@ -711,6 +722,16 @@ def run_stress_thread_bench(self, stress_cmd, duration=None, stats_aggregate_cmd
                                               self.outputdir,
                                               node_list=self.db_cluster.nodes)
 
+    @clean_resources_on_exception
```

Comment:
we don't need this decorator

Reply:
why it still used for cs-stress methods?

Reply:
legacy code, should be removed too

Reply:
I think you should handle it separately and remove all the usages of this decorator